features-json/push-api.json

```
@@ -298,8 +298,8 @@
       "15.2-15.3":"n #3",
       "15.4":"n #3",
       "15.5":"n #3",
-      "16.0":"y",
-      "TP":"n d #5"
+      "16.0":"y #6",
+      "TP":"n d #5 #6"
```

```suggestion
      "TP":"y #6"
```

Not sure if the corresponding version of TP is already out, but I believe we can be very sure it will match Safari 16 capabilities, so I would have changed that already 0:-)